src/sentry/search/events/datasets/base.py

```diff
@@ -55,3 +62,170 @@ def result_type_fn(
                 return argument.get_type(value)
 
         return result_type_fn
+
+
+def project_threshold_config_resolver_factory(
```

Comment:
Can we move this to a new file `src/sentry/search/events/datasets/function_aliases.py`